immersive-web-wg-charter.html

```diff
@@ -675,6 +675,16 @@ <h3 id="w3c-coordination">W3C Groups</h3>
             The Immersive Web Working Group will incubate and work on new model element proposal that are within 
             scope of WICG. 
           </dd>
+          
+          <dt><a href="https://www.w3.org/community/wicg/" target="_blank">Open Geospatial Consortium Community Group</a></dt>
```

Comment:
Is this https://www.w3.org/groups/wg/sdw ?
If OGC GeoPose SWG, https://www.ogc.org/projects/groups/geoposeswg , need to be listed a subsection below (external organization)

Reply:
@himorin as we are discussing [here ](https://github.com/w3c/immersive-web-wg-charter/issues/44), we want to make sure we have people working on [Geo Alignment](https://github.com/immersive-web/geo-alignment) repo, as part of the CG, to be picked up by WG in the future. Previously, we wanted to work with Geo Data group but since there was no work done on our side, there was nothing we can discuss. Does that make sense?
Also, is there a reason that we should add OGC instead of [SWG](https://www.w3.org/groups/wg/sdw)?

Reply:
Please have a look at the URL... (no preference between OGC or SWG)

Reply:
We will discuss it tomorrow and I'll add after. Thanks!

Reply:
I've add both OGC and geo-alignment repo. Please review and merge when you have a chance. Thanks